safehttp/default_dispatcher_test.go

```diff
@@ -51,11 +51,7 @@ func TestDefaultDispatcherValidResponse(t *testing.T) {
 						Parse("<h1>{{ . }}</h1>")))
 				var data interface{}
 				data = "This is an actual heading, though."
-				resp := safehttp.TemplateResponse{
-					Template: &t,
-					Data:     &data,
-				}
-				return d.ExecuteTemplate(w, resp)
+				return d.Write(w, safehttp.NewTemplateResponse(t, data, nil))
```

Comment:
`NewTemplateResponse` will be used *a lot* and it seems quite a long name to use.

What do you think about just `Tpl` or `TplResponse` or `Template` or other ways to shorten it?

Reply:
Is `TemplateResp` and `JSONResp` okay?

Reply:
Wasn't the idea to do:

```
func handler(w safehttp.ResponseWriter, req IncomingRequest) {
  // ...
  safehttp.ExecuteTemplate(w, tpl, data)
}
```